it-tests/Z_IntegrationsViewNewProject.test.ts

```diff
@@ -96,6 +96,7 @@ describe('Integrations View', function () {
 
 			// GAV selection
 			input = await InputBox.create(30_000);
+			await input.setText('com.acme:my-project:1.0-SNAPSHOT');
```

Comment:
I think it is more important to continue testing with the default value here. And have a set of unit tests for the various validation cases?